github-sync/github-data/users.yaml

```diff
@@ -1,9 +1,6 @@
 users:
   - username: A0su
```

Comment:
@haydentherapper @bobcallaway by removing the user from that team, meaning they don't have any other contribution for sigstore? should we keep the user as well or remove it from the org as well?

Reply:
ok, sounds good. I will do that in a follow-up to make more explicit